src/app/pages/network/interfaces/interfaces-list/interfaces-list.component.ts

```diff
@@ -22,6 +25,11 @@ export class InterfacesListComponent {
     message: T("Network connectivity will be interrupted. "),
   }
   protected hasDetails = true;
+  protected entityList: any;
+  protected checkChangesSubscription: any;
+  public hasPendingChanges = false;
+  pending_changes_text = T('You currently have pending network changes, do you wish to commit now?\
```

Comment:
As above.  Isn't this a repeated string?

Reply:
That should be removed now and only in the helptext file